### DISCUSSION TITLE
是否可以增加ChatGPT(Web)翻译服务的选项？（不是OpenAI API）

### DISCUSSION THREAD

**Discussion by 0xueping** [immersive-translate/immersive-translate#582] [Ideas]
### 新功能 | New Feature

你好！是否可以增加ChatGPT(Web)翻译服务的选项？注意：不是现有的OpenAI API翻译服务。

### 为什么您想要添加这些功能 | Why you want to add these features

原因是，实际上我不怎么用OpenAI API的方式，我主要还是用ChatGPT Plus每月$20/月订阅费的方式，每个月额度基本上也用不完。我希望沉浸式翻译能够调用我网页ChatGPT Plus（基于Web浏览器access token信息？？？），这样就不需要额外再去扣除一份OpenAI API的费用了。

**Comment by theowenyoung** [comment]
现在已支持，不过体验一般。因为web版的频率限制更严格。

可以在【开发者设置】 开启 beta 特性，即可看到。
